_posts/2015-02-03-8.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Creating a map"
+date:   2015-02-03 10:00:08
+answerrostag: "simulation_creat_map"
+track: [simulation]
+---
+
+***NOTE:*** *Some parts of this article duplicate information from [TurtleBot Simulation]({{ site.url }}{% post_url 2015-02-01-28 %}) post. We will merge them later.*
```

Comment:
There's a keyword `include` in jekyll which will let you include snippets. Documentation here: https://jekyllrb.com/docs/templates/#includes

This will let you have one file with content which is rendered in multiple places.